gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/validation/ListenerValidationServiceImpl.java

```diff
@@ -214,6 +215,12 @@ private void checkEntrypointConfiguration(final Entrypoint entrypoint) {
         entrypoint.setConfiguration(entrypointService.validateConnectorConfiguration(entrypoint.getType(), entrypointConfiguration));
     }
 
+    private void checkEntrypointListenerType(final ListenerType type, final ConnectorPluginEntity connectorPlugin) {
+        if (connectorPlugin.getSupportedListenerType() != null && type != connectorPlugin.getSupportedListenerType()) {
```

Comment:
same ignore validation if getSupportedListenerType is null. I don't know what to think. Maybe you will have an opinion ?¿

Reply:
🤷 If the connectorPlugin's supported listener type is null it means that the plugin is badly written, as we are expecting all entrypoint connector plugins to define a listener type 🤔 
For me, it should also throw an error, but this error should be thrown sooner while the plugin is loaded because it is not related to what is provided by the API publisher...